cmd/syncthing/main.go

```diff
@@ -783,20 +798,12 @@ func syncthingMain(runtimeOptions RuntimeOptions) {
 		}
 	}
 
-	myDev, _ := cfg.Device(myID)
-	l.Infof(`My name is "%v"`, myDev.Name)
-	for _, device := range cfg.Devices() {
-		if device.DeviceID != myID {
-			l.Infof(`Device %s is "%v" at %v`, device.DeviceID, device.Name, device.Addresses)
-		}
-	}
-
 	// Candidate builds always run with usage reporting.
 
 	if opts := cfg.Options(); build.IsCandidate {
 		l.Infoln("Anonymous usage reporting is always enabled for candidate releases.")
-		if opts.URAccepted != usageReportVersion {
-			opts.URAccepted = usageReportVersion
+		if opts.URAccepted != ur.UsageReportVersion {
+			opts.URAccepted = ur.UsageReportVersion
```

Comment:
ur.Version?
Also, perhaps ur -> reporting?

Reply:
ur.Version? Yes.  
reporting seems somewhat too generic, so I'd rather go with usagereporting, which seems a bit long. Not sure about this.

Reply:
Let's keep it as it is I guess.